packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -1179,6 +1180,11 @@ class C4DMastheadComposite extends HostListenerMixin(LitElement) {
         `;
   }
 
+  protected _renderCart() {
+    const { hasCart } = this;
+    return hasCart ? html`<c4d-masthead-cart></c4d-masthead-cart>` : undefined;
```

Comment:
I think we probably need a way to pass through a value to `link-label` from the `masthead-container`

Reply:
🤔 , if I look at the other icons, `<c4d-masthead-contact>` [does something similar](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/src/components/masthead/masthead-composite.ts#L1176-L1177) via `@property({ attribute: false })`. Want to do the same, or should be go with a reflected attribute?

Reply:
Yeah that seems like a good pattern to follow - we just need a way for use of the `masthead-container` to be able to pass in the translated value for the link label.

Reply:
Might be better as a follow up to address these hidden label translation issues across the <c4d-masthead-global-bar>` items.